base/applications/cmdutils/taskkill/taskkill.c

```diff
@@ -208,6 +210,71 @@ static BOOL get_process_name_from_pid(DWORD pid, WCHAR *buf, DWORD chars)
  *
  * A PID of zero causes taskkill to warn about the inability to terminate
  * system processes. */
+
+
+static BOOL get_pid_creation_time(DWORD pid, FILETIME *time)
+{
+    HANDLE process = INVALID_HANDLE_VALUE;
+    FILETIME t1 = { 0 }, t2 = { 0 }, t3 = { 0 };
+    BOOL result = TRUE;
+
+    process = OpenProcess(PROCESS_QUERY_INFORMATION, FALSE, pid);
+    if (!process)
+    {
+        result = FALSE;
+    }
+
+    if (!GetProcessTimes(process, time, &t1, &t2, &t3))
```

Comment:
How is this supposed to work if the process could not be opened?

Reply:
Sorry, my mistake. I will fix it.

Reply:
I fixed.

Reply:
The function is still called even if `process == NULL`. So I would suggest instead, that you just return FALSE above in case `OpenProcess()` failed.

Reply:
Thanks, I fixed.